src/en/help-azure.md

```diff
@@ -26,44 +27,53 @@ Before you can use Azure with Juju, you will need to import your Azure
 account credentials into Juju. Retrieving those credentials is easy, thanks to
 Microsoft's new [Azure Cloud Shell][azureshell].
 
-This can be installed using *[snap][snapcraft]*:
+Azure Cloud Shell can be installed using *[npm][npminfo]*. Enter the following
+command to install npm if it isn't already on your system:
 
 ```bash
-sudo snap install azure-cli --classic --edge
+sudo apt install -y nodejs-legacy npm
```

Comment:
Sorry I didn't see this happening earlier, but if we're updating these docs, please be aware that there's an entirely new Python-based Azure CLI (Azure CLI 2.0): https://github.com/Azure/azure-cli. This is the CLI recommended by Microsoft now, and supersedes the older NodeJS-based implementation.